charts/router/values.yaml

```diff
@@ -6,16 +6,6 @@ dhparam: ""
 # limits_cpu: "100m"
 # limits_memory: "50Mi"
 
-# Any custom router annotations(https://github.com/deis/router#annotations)
-# which need to be applied can be specified as key-value pairs under "deployment_annotations"
-deployment_annotations:
```

Comment:
Should we just comment out `deployment_annotations:` here rather than remove it?